### DISCUSSION TITLE
Place block for safe location if not find

### DISCUSSION THREAD

**Discussion by nivcoo** [BG-Software-LLC/SuperiorSkyblock2#480] [Pending Feature Requests]
Hello, i saw you edit the safe block location algorithme, but can you add block if for example my player break all block of his island (Now i've bedrock but i think it's good update because lot of my player want to remove this block)

**Comment by OmerBenGera** [comment]
It's an easy way to generate infinite blocks all the time.
Also, how would the plugin know where to place the block? it might override other blocks, etc.
It's a great feature on the papers, but to program such thing can cause issues, in my opinion at least.
If someone wants to develop such thing and make a PR, you are more than welcome to do so :)

**Reply by nivcoo** [reply]
If the plugin doesn't fond block i think the island is the void so you can place and check above the block if is air and place it. Yes we can generate infinite blocks but we can place customizable block like STONE/Cobble (Stone/cobble generator generate infinite stone)
